include/nbl/asset/IDescriptorSetLayout.h

```diff
@@ -467,7 +467,6 @@ class IDescriptorSetLayout : public IDescriptorSetLayoutBase
 				if ((b.type == IDescriptor::E_TYPE::ET_SAMPLER or b.type == IDescriptor::E_TYPE::ET_COMBINED_IMAGE_SAMPLER) and b.immutableSamplers)
 				{
 					const auto localOffset = m_immutableSamplerRedirect.getStorageOffset(typename CBindingRedirect::binding_number_t(b.binding)).data;
-					assert(bool(localOffset));
```

Comment:
why did you remove this?

Reply:
when `localOffset==0` (copying from the beginning) it aborts

Reply:
I don't know if that was the intent

Reply:
that would be because we have a `.data` there.

I know how to change it, my bug/typo

Reply:
fixed on `asset-conversion-v3` branch